pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio_test.go

```diff
@@ -607,3 +608,22 @@ func TestResourceBinPackingMultipleExtended(t *testing.T) {
 		})
 	}
 }
+
+// TODO compatibility test once the plugin args move to v1beta1.
+// UtilizationShapePoint and ResourceSpec fields of the plugin args struct are not annotated
+// with JSON tags in v1alpha2 to maintain backward compatibility with the args shipped with v1.18.
+// See https://github.com/kubernetes/kubernetes/pull/88585#discussion_r405021905
+
+func TestPluginArgsJSONEncodingIsCaseInsensitive(t *testing.T) {
+	rawArgs := &runtime.Unknown{Raw: []byte(`
+	{
+		"shape": [{"Utilization": 1, "Score": 1}, {"utilization": 2, "score": 2}],
+		"resources": [{"Name":"a","Weight":1},{"name":"b","weight":2}]
+	}
+`)}
+
+	args := &schedulerv1alpha2.RequestedToCapacityRatioArgs{}
+	if err := framework.DecodeInto(rawArgs, args); err != nil {
+		t.Fatalf("expected no error: %v", err)
```

Comment:
We don't just need to check for no errors, but that all values were set.

Reply:
Done